src/Fixer/Import/OrderedImportsFixer.php

```diff
@@ -251,4 +408,23 @@ private function getNewOrder(array $uses, Tokens $tokens)
 
         return $usesOrder;
     }
+
+    /**
+     * @param $indexes
+     *
+     * @return array
+     */
+    private function sortByAlgorithm($indexes)
+    {
+        switch ($this->sortAlgorithm) {
+            case self::SORT_LENGTH:
+                uasort($indexes, array($this, 'sortByLength'));
+                break;
+            default:
```

Comment:
awesome, so, the config value is validated and is always great.
and then, one will be playing around and may, even by accident, just do: `$this->sortAlgorithm = 'foobarbaz'` ;)